### ISSUE TITLE
Investigate colour contrast issues with the header component

### ISSUE THREAD

**Issue by 36degrees** [alphagov/govuk-frontend#1133]
## What

The blue (`#1d8feb`) used to represent the active state of links in header technically meets [WCAG 2.1 AA standards for contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) against the black (`#000000`), at a contrast ratio of 6.2:1, but has been flagged as potentially problematic. We should consider improving the contrast.

|     | Text | Large Text |
|-----|------|------------|
| AA  | ✔️   | ✔️         |
| AAA | ❌   | ✔️         |

## Why

This was flagged in an audit of the design system (5th June 2018):

> "The choice of colour used for the links within the header navigation may be problematic for some users" (DAC-USER-LV-GT-T22-01)

**Comment by kellylee-gds** [comment]
Closed issue as its in the done column on the Kanban board
